### PR TITLE
Removed commented code as per issue #5

### PR DESCRIPTION
**Pull request by TenTraicion**
I have removed the code as per the request of issue: #5

### CODE REVIEW

**Comment by lawgimenez** [comment]
Thank you! Will compile today.

**Comment by TenTraicion** [comment]
@lawgimenez may I know why this PR was closed instead of merge? was there any issue with the PR?

File: app/src/androidTest/java/com/example/kg_cai/ExampleInstrumentedTest.java [hunk 1]
```diff
@@ -10,11 +10,6 @@
 
 import static org.junit.Assert.*;
 
-/**
```
Comment (lawgimenez) [inline app/src/androidTest/java/com/example/kg_cai/ExampleInstrumentedTest.java:13]: Sorry, I did not mean to include this. Just the unused code blocks in //

**Comment by lawgimenez** [comment]
Will merge this manually later.

**Review by lawgimenez — APPROVED** [review]

**Comment by TenTraicion** [comment]
as the issue no longer exist, and the PR is left alone, i am closing this. Happy Coding!

**Pull request closed without merging**